docs/config.rst

```diff
@@ -62,13 +111,76 @@ Java Kafka clients.
   * Default: ""
   * Importance: medium
 
+``simpleconsumer.cache.max.records``
+  Maximum number of records that can be stored for a specific topic-partition combination. Records with higher offsets replace records with lower ones Must be greater that 0.
+
+  * Type: int
+  * Default: 1000
+  * Importance: medium
+
+``simpleconsumer.max.caches.num``
```

Comment:
This configuration doesn't seem too useful. I guess the idea is to bound the total memory devoted to the cache, but that's difficult to do anyway knowing the size of the cached records. In practice, it seems simpler to use only per-partition cache settings and let users scale the number of proxy instances as needed. What do you think?